README.md

```diff
@@ -141,10 +141,20 @@ There are several ways to fix this:
     * This behavior can be disabled by setting `ignoreAuthFailures: false` in the config file.
 
 1. How does `xrefcheck` handle redirects?
-    * Permanent redirects (i.e. 301 and 308) are reported as errors.
-    * Temporary redirects (i.e. 302, 303 and 307) are assumed to be valid.
-
-1. How does `xrefcheck` handle localhost links?
+    * The default rules are as follows:
+      * Permanent redirects (i.e. 301 and 308) are reported as errors.
+      * Temporary redirects (i.e. 302, 303 and 307) are assumed to be valid.
+    * The redirect rules can be specified with the `externalRefRedirects` within `networking`, which accepts an array of
+      rules like `{to: "^http://.*", on: temporary, outcome: follow }`. The rule applied is the first one that matches with
```

Comment:
It remains to decide if the rule that applies is the first or the last that matches.